systemd/systemd.go

```diff
@@ -1064,9 +1112,21 @@ LazyUnmount=yes
 WantedBy=multi-user.target
 `
 
-func writeMountUnitFile(snapName, revision, what, where, fstype string, options []string) (mountUnitName string, err error) {
-	content := fmt.Sprintf(mountUnitTemplate, snapName, revision, what, where, fstype, strings.Join(options, ","))
-	mu := MountUnitPath(where)
+const (
+	snappyCreatorModule = "X-SnappyCreatorModule"
```

Comment:
naming consideration, we shouldn't introduce new things using Snappy in their name.  X-SnapdOrigin perhaps?